rabbit-escape-render/src/rabbitescape/render/SpriteAnimator.java

```diff
@@ -57,7 +56,7 @@ public SpriteAnimator(
         this.animationCache = animationCache;
     }
 
-    public List<Sprite> getSprites( int frameNum )
+    public List<Sprite> getSprites( int frameNum, WaterDynamics waterDynamics )
```

Comment:
Fossil code accident. 

i am now thinking droplets for falling water could just be transparent circles, so this wont be needed. (it might have been if they were sprites).

removed.